packages/timere/timere.0.1.3/opam

```diff
@@ -0,0 +1,51 @@
+opam-version: "2.0"
+synopsis: "OCaml time handling and reasoning library"
+description: """
+Features:
+
+- Timestamp and date time handling with platform independent time zone support
+
+  - Subset of the IANA time zone database is built into this library
+
+- Reasoning over time intervals via `timere` objects/expressions, examples:
+
+  - Pattern matching time and intervals. These work across DST boundaries.
+ 
+  - Intersection and union
+
+  - Chunking at year or month boundary, or in fixed sizes
+
+  - Evaluate (sub)expressions with a different time zone (e.g. intersection of 9am to 5pm of Sydney and 9am to 5pm of New York)
+"""
+maintainer: "Darren Ldl <darrenldldev@gmail.com>"
+authors: "Daypack developers"
+license: "MIT"
+homepage: "https://github.com/daypack-dev/timere"
+bug-reports: "https://github.com/daypack-dev/timere/issues"
+dev-repo: "git+https://github.com/daypack-dev/timere"
+depends: [
+  "ocaml" {>= "4.06.1"}
+  "bisect_ppx" {dev & >= "2.5.0"}
+  "dune" {>= "1.11.0"}
```

Comment:
```suggestion
  "dune" {>= "2.7.0"}
```
The dune-project file requires 2.7, I think it is due to the new bisect_ppx support

Reply:
@darrenldl can you upstream this change to both the opam files?